runtime/service.go

```diff
@@ -171,6 +173,11 @@ func NewService(shimCtx context.Context, id string, remotePublisher shim.Publish
 		}
 	}()
 
+	tempDir, err := ioutil.TempDir("", "stub-drives")
```

Comment:
I just realized we could instead use the `vmDir` to store all these instead of `/tmp`. That way, whenever we cleanup the `vmDir` we just inherently also cleanup any leftover stub drives. Just less places to manage and think about.